contributors/guide/owners.md

```diff
@@ -86,6 +86,28 @@ Instead, set a `.*` key inside `filters` (as shown in the previous example).
 **WARNING**: The `approve` plugin [does not currently respect `filters`][test-infra-7690].
 Until that is fixed, `filters` should only be used for the `labels` key (as shown in the above example).
 
+
+#### Emeritus
+
+It is inevitable, but there are times when someone may shift focuses, change jobs or step away from
+a specific area in the project for a time. These people may be domain experts over certain areas
+of the codebase, but can no longer dedicate the time needed to handle the responsibilities of
+reviewing and approving changes. They are encouraged to add themselves as an _"emeritus"_ approver
```

Comment:
What more does this accomplish than what we can do already:
```yaml
approvers:
# - fejta (emeritus)
- mrbobbytables
- nikhita
- spiffxp
```

Reply:
It has been used informally in a few places. Trying to put some language in here for wider use and encourage folks to self-remove; along with adding a potential avenue for future tooling.